airflow/contrib/utils/sendgrid.py

```diff
@@ -63,6 +64,12 @@ def send_email(to, subject, html_content, files=None,
     mail.add_personalization(personalization)
     mail.add_content(Content('text/html', html_content))
 
+    # Add custom_args to personalization if present
+    pers_custom_args = kwargs.get('personalization_custom_args', None)
+    if isinstance(pers_custom_args, dict):
```

Comment:
nit: should this be if pers_custom_args?

Reply:
My arbitrary decision to ignore invalid parameters instead of letting error occur later on